src/api/storage-pools.tsx

```diff
@@ -381,3 +381,38 @@ export const migrateStorageVolume = (
       .catch(reject);
   });
 };
+
+// Including project and target params if they did not change from source configs breaks the API call.
+// Therefore, we only include them if they are different from the source configs, that's why both project and target are optional inputs
+export const duplicateStorageVolume = (
+  volume: Partial<LxdStorageVolume>,
+  pool: string,
+  project?: string,
+  target?: string,
+): Promise<LxdOperationResponse> => {
+  return new Promise((resolve, reject) => {
+    const url = new URL(
+      `/1.0/storage-pools/${pool}/volumes/custom`,
+      window.location.origin,
+    );
```

Comment:
```suggestion
    const url = new URL(`/1.0/storage-pools/${pool}/volumes/custom`);
```
I think this is not needed.

Reply:
Would you be able to give me a reproducer for the error you are seeing? I can replicate it locally or on the demo server. 

Edit: if I removed `window.location.origin` from the `URL` construction, then I see the above error :thinking:

Reply:
Doh, my bad. then all good as it is. Sorry for the confusion.

Reply:
all good!